config/initializers/types.rb

```diff
@@ -1,3 +1,2 @@
+ActiveModel::Type.register(:value_object, ValueObjectType)
 ActiveRecord::Type.register(:value_object, ValueObjectType)
```

Comment:
I _think_ we need both since we definining attributes (of type `:value_object`) for AR instances (like `Player`) and for objects that subclass `ValueObject` which builds upon `ActiveModel::Model` and `ActiveModel::Attributes`.

_I might be wrong though_

Reply:
Good point 🤔 I'm not sure to be honest. But could we evaluate this by just using `ActiveModel::Type` and see if everything still works?

Reply:
I tried this and got some _unknown attribute error_. pls try it out.

Reply:
Yep, a nasty error came up. I don't think we should put any time in this. It works with both registrations and that's fine with me.